assets/src/editor/blocks/landing-page-hero/index.js

```diff
@@ -16,6 +20,7 @@ import ImageFilter, { DEFAULT_IMAGE_FILTER } from '../../components/image-filter
 import ImagePicker from '../../components/image-picker';
 import blockStyles, { applyDefaultStyle } from '../../helpers/block-styles';
 import './style.scss';
+import { SelectControl, Panel, PanelBody, } from '@wordpress/components';
```

Comment:
Stylistically, this should come right after the imports from `@wordpress/block-editor` to follow the style guidelines for import order.